presto-docs/src/main/sphinx/release/release-0.236.rst

```diff
@@ -0,0 +1,26 @@
+=============
+Release 0.236
+=============
+
+General Changes
+_______________
+* Fix query failure due to searched case expression with large number of cases.
+* Improve experience by warning user of poorly performant ``UNION DISTINCT`` usage.
+
+Verifier Changes
+________________
+* Add ``ErrorType`` of query failures to verification outputs.
+* Add support to auto-resolve result mismatch for structured-type columns.
+* Add support to auto-resolve result mismatch in case the query uses functions to be ignored.
```

Comment:
Doesn't this section go to the end?

Reply:
Yep, let's move `Hive Changes` upward. We can also adjust the release notes tool.